packages/aws-cdk-lib/aws-stepfunctions/lib/state-machine.ts

```diff
@@ -60,8 +60,10 @@ export enum LogLevel {
 export interface LogOptions {
   /**
    * The log group where the execution history events will be logged.
+   *
+   * @default No log group. Required if your log level is not set to OFF.
    */
-  readonly destination: logs.ILogGroup;
+  readonly destination?: logs.ILogGroup;
```

Comment:
will this not be a breaking change for existing usage? example:
```
const logOptions: LogOptions = {
  destination: <LogGroup>
}

const stateMachine: StateMachine = new StateMachine(parent, 'ID', {
  logs: logOptions,
  ...
}
... 

stateMachine.logs.destination.logGroupName // Error
stateMachine.logs.destination!.logGroupName // Change needed
```
Some user(s) might be utilising `ILogGroup`'s public properties from `logOptions` which will now cause issues because now such users will need to make non-null assertions before using such properties (e.g. with `!`)

Reply:
According to the [jsii-diff documentation](https://github.com/aws/jsii/blob/main/packages/jsii-diff/BREAKING_CHANGES.md#making-properties-optional):
> You are allowed to make _inputs_ optional

So i think this change is not a breaking change.